boards/arm/adafruit_trinket_m0/adafruit_trinket_m0.dts

```diff
@@ -6,6 +6,7 @@
 
 /dts-v1/;
 #include <atmel/samd21.dtsi>
+#include <atmel/samd2xx18.dtsi>
```

Comment:
This should follow same solution of #61744 which is include main header inside `samd2xx18.dtsi`